src/viam/module/module.py

```diff
@@ -132,6 +134,7 @@ async def add_resource(self, request: AddResourceRequest):
         model = Model.from_string(config.model, ignore_errors=True)
         creator = Registry.lookup_resource_creator(subtype, model)
         resource = creator(config, dependencies)
+        logging.getLogger(request.config.name)
```

Comment:
What is this for?

Reply:
Thanks for catching, it's an artifact from testing and trying something else.